docs/plugins/repository-s3.asciidoc

```diff
@@ -74,71 +78,79 @@ contents, will utilize the latest settings from the keystore. Any existing `s3`
 repositories, as well as any newly created ones, will pick up the new values
 stored in the keystore.
 
-NOTE: In progress snapshot/restore tasks will not be preempted by a *reload*
-of the client's secure settings. The task will complete using the client as it
-was built when the operation started.
+NOTE: In-progress snapshot/restore tasks will not be preempted by a *reload* of
+the client's secure settings. The task will complete using the client as it was
+built when the operation started.
 
 The following list contains the available client settings. Those that must be
 stored in the keystore are marked as "secure" and are *reloadable*; the other
 settings belong in the `elasticsearch.yml` file.
 
 `access_key` ({ref}/secure-settings.html[Secure])::
 
-    An s3 access key. The `secret_key` setting must also be specified. 
+    An S3 access key. The `secret_key` setting must also be specified.
 
 `secret_key` ({ref}/secure-settings.html[Secure])::
 
-    An s3 secret key. The `access_key` setting must also be specified. 
+    An S3 secret key. The `access_key` setting must also be specified.
 
 `session_token`::
-    An s3 session token. The `access_key` and `secret_key` settings must also
+    An S3 session token. The `access_key` and `secret_key` settings must also
     be specified. (Secure)
 
 `endpoint`::
 
-    The s3 service endpoint to connect to. This will be automatically
-    figured out by the s3 client based on the bucket location, but
-    can be specified explicitly. See http://docs.aws.amazon.com/general/latest/gr/rande.html#s3_region.
+    The S3 service endpoint to connect to. This defaults to `s3.amazonaws.com`
+    but the
+    http://docs.aws.amazon.com/general/latest/gr/rande.html#s3_region[AWS
+    documentation] lists alternative S3 endpoints. If you are using an
+    S3-compatible service such as https://minio.io[Minio] then you should set
```

Comment:
I particularly wonder whether we should mention Minio specifically here. On the one hand, Minio is the only "S3-compatible" service that we test against so perhaps that deserves a mention. On the other hand, mentioning a specific service seems a bit weird.

Reply:
I'm fine with mentioning Minio here.